students/hyska_monika/lesson_05_ Lists_list_comprehensions/SwapTheColumns.py

```diff
@@ -0,0 +1,17 @@
+# Function return/print array after swap columns
+from create_lists import array_2dem_random
+
+
+def swap_columns(mylist):
+    i = int((input("First column to swap: ")))
+    j = int((input("Second column to swap: ")))
+    for elem in range(len(mylist)):
+        temp = mylist[elem][i]
+        mylist[elem][i] = mylist[elem][j]
+        mylist[elem][j] = temp
```

Comment:
There is an easier way in python: `mylist[elem][i], mylist[elem][j] = mylist[elem][j], mylist[elem][i]`